### PR TITLE
Chore/update branching doc

### PR DESCRIPTION
**Pull request by saltcod**

### CODE REVIEW

File: apps/docs/pages/guides/platform/branching.mdx [hunk 1]
```diff
@@ -44,35 +47,36 @@ Each Supabase project can consist of multiple Preview Branches. Each Preview Bra
...
 
 ### Branching with GitHub
 
 Supabase Branching uses the Supabase GitHub integration to read files from your GitHub repository. With this integration, Supabase watches all commits, branches, and pull requests of your GitHub repository.
 
-One of your Git branches is your Production Branch. This should also be your Supabase project's Production Branch.
+In Git, you have a Production Branch (typically this is `main`, `master`, `prod`, etc). This should also be your Supabase project's Production Branch.
 
-For every Git Branch in your GitHub repository, you may have a corresponding Preview Branch. Each time a new Preview Branch is created, the migrations in the Git branch of that Preview Branch are run on the Preview Branch.
+You can create a corresponding Preview Branch for any Gith Branch in your repository. Each time a new Preview Branch is created, the migrations in the Git branch of that Preview Branch are run on the Preview Branch. Just like in Git, a Preview Branch is an exact copy of the Production Branch at the time of creation.
```
Comment (MildTomato) [inline apps/docs/pages/guides/platform/branching.mdx:79]: I don't capitalize branch in 'Git branch' - Only 'Preview Branch' as it's one of our features.

File: apps/docs/pages/guides/platform/branching.mdx [hunk 2]
```diff
@@ -169,7 +172,7 @@ Follow the instructions below **before turning on Supabase branching**.
 
 If your migration files are up to date with your production database, you don’t need to do anything. You can now turn on branching.
 
-##### If you don’t have migration files in GitHub:
+##### If you don't have migration files in GitHub:
```
Comment (MildTomato) [inline apps/docs/pages/guides/platform/branching.mdx:175]: huh? weird

File: apps/docs/pages/guides/platform/branching.mdx (hunk 1 above)
Comment (MildTomato) [reply apps/docs/pages/guides/platform/branching.mdx:79]:
```suggestion
You can create a corresponding Preview Branch for any Git branch in your repository. Each time a new Preview Branch is created, the migrations in the Git branch of that Preview Branch are run on the Preview Branch. Just like in Git, a Preview Branch is an exact copy of the Production Branch at the time of creation.
```

> Just like in Git, a Preview Branch is an exact copy of the Production Branch at the time of creation.

Not sure this is true for everyone. It's generally the working practice though for many though.

File: apps/docs/pages/guides/platform/branching.mdx [hunk 3]
```diff
@@ -44,35 +47,36 @@ Each Supabase project can consist of multiple Preview Branches. Each Preview Bra
   <figcaption>Each branch has a separate Supabase instance.</figcaption>
 </figure>
 
-Conceptually, Supabase Branching works like Git branching, in which you usually have a designated branch for Production, and then you use other branches to propose changes that will be merged into your Production branch.
+Supabase branching works like Git branching, with one main Production branch and others for proposing and merging changes into it.
```
Comment (charislam) [inline apps/docs/pages/guides/platform/branching.mdx:50]:
```suggestion
Supabase branching works like Git branching, with one main Production branch. Preview Branches branch off your Production branch and contain proposed changes. These changes can then be merged into Production. 
```

File: apps/docs/pages/guides/platform/branching.mdx [hunk 4]
```diff
@@ -44,35 +47,36 @@ Each Supabase project can consist of multiple Preview Branches. Each Preview Bra
...
 
-When you create a Preview Branch (which will be merged off from your Production branch), the SQL migration files from your Production branch are run on the new Preview Branch. The Preview Branch then reflects your current Production branch schema.
+When you create a Preview Branch (branched off your Production branch), the SQL migration files from your Production branch are run on the new Preview Branch. The Preview Branch then reflects your current Production branch schema.
```
Comment (charislam) [inline apps/docs/pages/guides/platform/branching.mdx:52]:
```suggestion
When you create a Preview Branch, the SQL migration files from your Production branch are run on the new Preview Branch. The Preview Branch then reflects your current Production branch schema.
```

File: apps/docs/pages/guides/platform/branching.mdx [hunk 5]
```diff
@@ -155,9 +158,9 @@ Push your new project to a GitHub repo. For more information, see the GitHub gui
 
 #### Set up an existing project for branching
 
-Before enabling branching on an existing project, make sure your database-related files are up to date. You need a correct snapshot of your existing database, so newer migration files have a foundation to run on top of.
+Before enabling branching on an existing project, make sure your database-related files are up to date. You need a current snapshot of your existing database, so newer migration files have a foundation to run on top of.
 
-<Admonition type="caution">
+<Admonition type="caution" label="Get setup before turning on branching">
```
Comment (charislam) [inline apps/docs/pages/guides/platform/branching.mdx:163]:
```suggestion
<Admonition type="caution" label="Get set up before turning on branching">
```

**Review by charislam — APPROVED** [review]
Thanks Terry! Minor suggestions

**Pull request merged**